src/main/java/com/kodilla/ecommercee/dto/UserDto.java

```diff
@@ -1,4 +1,21 @@
 package com.kodilla.ecommercee.dto;
 
+import com.kodilla.ecommercee.domain.UserKey;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
 public class UserDto {
+
+    private long id;
+    private String username;
+    private boolean status;
+    private UserKey userKey;
```

Comment:
wiem, że to nie encja, ale przekazujesz obiekt domenowy a nie Dto, które mogą się różnić polami - Dto może mieć ich mniej, bo niektórych nie chcemy wysyłać na zewnątrz
możesz tutaj umieścić UserKeyDto - wymaga to zmian w mapperze